keybord.js

```diff
@@ -0,0 +1,104 @@
+class keybord {
+    textarea;
+
+    init(keyLayout) {
+        const insertLineBreak = ["9", "Shift", "Ctrl", "capslock"];
+        let keyElement;
+        this.wrapper = document.createElement('div');
+        this.wrapper.setAttribute('class', 'wrapper');
+        this.keyBlock = document.createElement('div');
+        this.keyBlock.setAttribute('class', 'keyboard');
+        this.keyBlock.setAttribute('id', 'keyboard');
+        this.textArea = document.createElement('textarea');
+        this.textArea.setAttribute('rows', '15');
+        this.wrapper.appendChild(this.textArea);
+        for (let i = 0; i < keyLayout.length; i++) {
```

Comment:
`keyLayout` array looks like configuration. It's not a very good idea to mutate it later. Please don't do it.